content/security/for-admins/hardened-desktop/settings-management/_index.md

```diff
@@ -11,11 +11,11 @@ aliases:
 >
 >Settings Management is available to Docker Business customers only.
 
-Settings Management is a feature that helps admins to control certain Docker Desktop settings on client machines within their organization.
+Settings Management is a feature that helps admins to control certain Docker Desktop settings on client machines within their organization. It is designed specifically for organizations who don’t give developers root access to their machines.
```

Comment:
**question: (non-blocking)** Is `root access` the right term to use here? In Windows land, they don't have a concept of `root`. It's usually something like `administrator` or `elevated`?